genie-core/src/main/java/com/netflix/genie/core/services/impl/LocalJobRunner.java

```diff
@@ -203,6 +215,38 @@ public void submitJob(
         }
     }
 
+    private void createInitFailureDetailsFile(final String id, final Exception e) {
+        final long start = System.nanoTime();
+        try {
+            final File jobDir = new File(this.baseWorkingDirPath.getFile(), id);
+            if (jobDir.exists()) {
+                final File detailsFile = new File(jobDir, JobConstants.GENIE_INIT_FAILURE_MESSAGE_FILE_NAME);
+                final boolean detailsFileExists = !detailsFile.createNewFile();
+                if (detailsFileExists) {
+                    log.warn("Init failure details file exists");
+                }
+                try (final Writer writer = new OutputStreamWriter(new FileOutputStream(detailsFile), "UTF-8")) {
```

Comment:
The only constant I'm seeing is: https://github.com/Netflix/genie/blob/3.0.x/genie-web/src/main/java/com/netflix/genie/web/resources/handlers/GenieResourceHttpRequestHandler.java#L55